packages/garbo/garbo_choice.ts

```diff
@@ -0,0 +1,51 @@
+import { availableChoiceOptions, runChoice } from "kolmafia";
+
+type DartPriority = {
+  [key: string]: number;
+};
+
+export function main() {
+  const dartPriority: DartPriority = {
+    "Throw a second dart quickly": 60,
+    "Deal 25-50% more damage": 800,
+    "You are less impressed by bullseyes": 10,
+    "25% Better bullseye targeting": 20,
+    "Extra stats from stats targets": 40,
+    "Butt awareness": 30,
+    "Add Hot Damage": 1000,
+    "Add Cold Damage": 1000,
+    "Add Sleaze Damage": 1000,
+    "Add Spooky Damage": 1000,
+    "Add Stench Damage": 1000,
+    "Expand your dart capacity by 1": 50,
+    "Bullseyes do not impress you much": 9,
+    "25% More Accurate bullseye targeting": 19,
+    "Deal 25-50% extra damage": 10000,
+    "Increase Dart Deleveling from deleveling targets": 100,
+    "Deal 25-50% greater damage": 10000,
+    "25% better chance to hit bullseyes": 18,
+  };
```

Comment:
If this is going to be a bunch of constants, don't define it inside the scope of the main function, just put this in the top level

Reply:
But also i wouldn't have this be a bunch of constants (or at least not _these_ constants. Honestly it would be easier to have an array that's naturally sorted by priority, with the highest-value dart perks in the front and the lowest in the back